### PR TITLE
Adding Org chart representation on user view

### PR DESCRIPTION
**Pull request by veenone**
# Description

adding new tab in user view to display the position of the user within the department relative to his / her n-1 and n+1.
The purpose is to give additional view on the users relation within the department.
Currently we have manager which can  be associated with a user but we can't see the overview of the user's position within the department. 
Now we can see it 😃 

this implementation based on [this project](https://github.com/dabeng/OrgChart)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Test if data reflected properly according to the mapping with the managers

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* PHP version:
* MySQL version
* Webserver version
* OS version

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# Screenshot

## New tab called org chart in user view
image

## Org chart of the respective users
- if user doesn't have manager, he / she will be at the top level
- if user is a manager, his / her direct report will be displayed as well
- Chart can be exported as PNG file
image

## Org chart color scheme follows active theme
image
image
image
image

### CODE REVIEW

File: app/Http/Controllers/Users/UsersController.php [hunk 1]
```diff
@@ -671,4 +779,17 @@ public function sendPasswordReset($id)
...
+     * get user org list with input user ID is the manager
+     *
+     * @author A. Rahardianto
+     * @since [v6.1.0]
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function getUserOrg($id)
+    {
+
+        return response()->json(Helper::formatStandardApiResponse('error', null, $assetMaintenance->getErrors()));
```
Comment (snipe) [inline app/Http/Controllers/Users/UsersController.php]: I'm not sure what `$assetMaintenance->getErrors())` would be doing here?

**Comment by snipe** [comment]
Hey there - thanks for this :) This adds a buttload of new javascript that is used only for this one purpose. We've never (or at least not in recent memory) had anyone ask for this feature, so I'm curious whether this is something the majority of the users would even be interested in.

File: app/Http/Controllers/Users/UsersController.php (hunk 1 above)
Comment (veenone) [reply app/Http/Controllers/Users/UsersController.php]: oops, it's a garbage code. I forgot to remove these lines.
I will remove it

**Comment by veenone** [comment]
Hi @snipe, you're right I don't think there's a request on this feature, however I thought this can be an interesting implementation especially for admin to have small visual cues regarding a position of a user relatively within the organization. Since manager & title fields already available before, I was thinking that I can make use these fields into something more useful by reflecting the usage of them between users in a visual representation :)

This implementation based on [(this project)](https://github.com/dabeng/OrgChart) which I thought quite interesting feature to have here. However if you think the javascript is too overwhelming, let me know if I can trim it down for minimal usage only

**Pull request closed without merging**